pynitrokey/cli/nk3/piv.py

```diff
@@ -0,0 +1,739 @@
+import datetime
+import sys
+from typing import Optional, Sequence
+
+import click
+import cryptography
+from asn1crypto import x509
+from asn1crypto.core import ParsableOctetString
+from asn1crypto.csr import CertificationRequest, CertificationRequestInfo
+from asn1crypto.keys import PublicKeyInfo
+from click_aliases import ClickAliasedGroup
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import ec, rsa
+from cryptography.hazmat.primitives.serialization import Encoding
+
+from pynitrokey.cli.nk3 import nk3
+from pynitrokey.helpers import check_experimental_flag, local_critical, local_print
+from pynitrokey.nk3.piv_app import PivApp, find_by_id
+from pynitrokey.tlv import Tlv
+
+
+@nk3.group(cls=ClickAliasedGroup)
```

Comment:
AFAIS not necessary here, probably copied from secrets.

Reply:
Removed in the squash.